### DISCUSSION TITLE
Welcome to nwxtregress Discussions!

### DISCUSSION THREAD

**Discussion by JanDitzen** [JanDitzen/nwxtregress#1] [Announcements]
Any questions or comments are welcome here.

**Comment by baishengcui** [comment]
Dear Ditzen!


     When I use you Stata code , namely nwxtregress to run your example in help files. I always face an error.

The error results are as follows:


nwxtregress cap_cons compensation net_surplus, dvarlag(WSpmat) seed(1234)

This is an alpha version! Results may change.

Initialise Grid (200)
----+--- 10 --+--- 20 --+--- 30 --+--- 40 --+--- 50   %
..................................................   50
..................................................  100
Griddy Gibbs (2000)
----+--- 10 --+--- 20 --+--- 30 --+--- 40 --+--- 50   %
          mm_colrunsum():  3001  expected 1 arguments but received 3
          GriddyGibbsi():     -  function returned error
           GriddyGibbs():     -  function returned error
           estimatesar():     -  function returned error
                 <istmt>:     -  function returned error

Can you give me a help?

Thank you very much!

**Reply by JanDitzen** [reply]
Dear Bai,
thank you very much for your interest in nwxtregress. 

The error is caused by a program from moremata, mm_colrunsum(). As a first step, can you please check if you have installed the latest version of moremata by typing ```ssc install moremata``` in Stata. Please let me know if this solves the problem.

Thanks,
Jan

**Comment by baishengcui** [comment]
Dear Dizten,




   The moremat is the lastest version.


But the same error will still occur.  Please see the attachment file in detail!


Thank you very much!!!















--

Bai-Sheng CUI,PH.D
__________________________________
School of Finance and Business
Shanghai Normal University
100 Guilin Rd., Shanghai 200234
Tel: 86-21-64323361 (O)  13916936827

***@***.***




At 2021-08-28 22:46:35, "Jan Ditzen" ***@***.***> wrote:

Dear Bai,
thank you very much for your interest in nwxtregress.

The error is caused by a program from moremata, mm_colrunsum(). As a first step, can you please check if you have installed the latest version of moremata by typing ssc install moremata in Stata. Please let me know if this solves the problem.

Thanks,
Jan

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by JanDitzen** [reply]
Can you please upload the attachment again, I cant see it.
Thanks

**Comment by baishengcui** [comment]
Dear Ditzen,

      Today, the nwxtregress works correctly, although I haven't made any updates.

I'll try to make some new applications. I'll ask you if I have any questions.

Thanks.

Baisheng

**Reply by JanDitzen** [reply]
Great, many thanks for the feedback. Out of interest two questions: a) which Stata version do you use, and b) did you restart Stata after you installed moremata?

**Comment by baishengcui** [comment]
Dear Ditzen,


 Thank you for your help.


The Stata Version is 17.0.


I restarted Stata after I installed moremata.


Thanks.














--

Bai-Sheng CUI,PH.D
__________________________________
School of Finance and Business
Shanghai Normal University
100 Guilin Rd., Shanghai 200234
Tel: 86-21-64323361 (O)  13916936827

***@***.***




At 2021-08-30 14:56:34, "Jan Ditzen" ***@***.***> wrote:

Great, many thanks for the feedback. Out of interest two questions: a) which Stata version do you use, and b) did you restart Stata after you installed moremata?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
